### PR TITLE
Add cumsum function

### PR DESCRIPTION
**Pull request by ronekko**
I implemented `F.cumsum` as `FunctionNode`.

### CODE REVIEW

File: chainer/functions/math/cumsum.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+def cumsum(x, axis=None):
+    """Cumulative sum of array elements over a given axis.
+
+    Args:
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+           :class:`cupy.ndarray`):
+            Elements to calculate the cumulative sum.
+        axis (int, optional):
```
Comment (unnonouno) [inline chainer/functions/math/cumsum.py]: `int or None`

**Comment by unnonouno** [comment]
jenkins test this please

**Comment by ronekko** [comment]
Fixed.

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**